backend/metamodel/syson-sysml-metamodel/pom.xml

```diff
@@ -76,6 +76,16 @@
 			<artifactId>java-uuid-generator</artifactId>
 			<version>4.3.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.14.0</version>
```

Comment:
Please remove version tag (it will use the version associated with the version of spring boot used by SysON) unless there is a good reason.

Reply:
Ok